src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2991,17 +2993,21 @@ float Creature::GetAggroRange(Unit const* target) const
     if (aggroRate == 0)
         return 0.0f;
 
-    uint32 targetLevel = target->getLevelForTarget(this);
-    uint32 myLevel = getLevelForTarget(target);
-    int32 levelDiff = int32(targetLevel) - int32(myLevel);
+    uint32 creatureLevel = target->getLevelForTarget(this);
+    uint32 playerLevel  = getLevelForTarget(target);
```

Comment:
if the function getLeveForTarget has the right return type you can use `auto` instead of `uint32` in these two lines

Reply:
fixed